src/api/app/views/webui/users/tokens/new.html.haml

```diff
@@ -0,0 +1,84 @@
+- operation_options = Token::OPERATIONS.map { |a| "Token::#{a}".constantize.token_name }
```

Comment:
workflow only if you are in beta.

Reply:
I changed the flag checked from `user_profile_redesign` to `trigger_workflow`. I also updated the PR description. @hennevogel please review again.